prow/spyglass/gcsartifact_fetcher.go

```diff
@@ -118,17 +118,23 @@ func (af *GCSArtifactFetcher) artifacts(key string) ([]string, error) {
 		Versions: false,
 	}
 	objIter := bkt.Objects(context.Background(), &q)
-	for {
+	wait := []time.Duration{1, 2, 4, 8, 16, 32, 64, 128, 256, 512}
```

Comment:
Attempting 10 times in a second seems a little bit aggressive...

Reply:
I took off some of the shorter waits and added some longer waits. Didn't want to make the user wait _too_ long though, because this is blocking the page load